server/src/immich/config/asset-upload.config.ts

```diff
@@ -49,25 +49,74 @@ export const multerUtils = { fileFilter, filename, destination };
 
 const logger = new Logger('AssetUploadConfig');
 
+const validMimeTypes = [
+  'image/avif',
+  'image/dng',
+  'image/gif',
+  'image/heic',
+  'image/heif',
+  'image/jpeg',
```

Comment:
I believe there are cases with `image/jpg` as well, right?

Reply:
No, that is not a valid mime type.